### DISCUSSION TITLE
Snippet script for TS

### DISCUSSION THREAD

**Discussion by Barbapapazes** [nuxtrdev/nuxtr-vscode#73] [Ideas]
Hello,

Could be a great addition to have a snippet for `<script lang="ts" setup>`!

**Comment by adhamfarrag** [comment]
How did we miss it! :) 

Added and should be published with next version!

![2023-10-22 at 12  32 56@2x](https://github.com/nuxtrdev/nuxtr-vscode/assets/23562275/2c792370-2987-4645-92b4-b690dc98e5c2)
